doc/source/whatsnew/v1.0.0.rst

```diff
@@ -229,6 +229,7 @@ Indexing
 - Bug in reindexing a :meth:`PeriodIndex` with another type of index that contained a `Period` (:issue:`28323`) (:issue:`28337`)
 - Fix assignment of column via `.loc` with numpy non-ns datetime type (:issue:`27395`)
 - Bug in :meth:`Float64Index.astype` where ``np.inf`` was not handled properly when casting to an integer dtype (:issue:`28475`)
+- Bug in :meth:`MultiIndex.get_loc` can't find ``np.nan`` when input is NA value (:issue:`19132`)
```

Comment:
Bug in ..... with a null value as input

Reply:
do we have a user visible effect of this? e.g. in actually indexing a Series/DataFrame.

Reply:
@jreback Well.... .loc can find nan with a null value as input. Also .get_loc can find values which include nan. But only when input is a null value, can't find indexer. So i think working .get_loc with a null value correctly also makes sense.

Reply:
sure, but ``.get_loc`` is not a very visible user visible function, what I mean is docs ``.loc[]`` on something in particular now work when it did not before?

Reply:
I change whatsnew. I try to pass on what i intend more clearly. Okay I understand. I agree that `get_loc` is not a very visible user visible function. But eventually, `get_loc` is used for index object not for Series/DataFrame. What i concern is explanation with indexing a Sereis/DataFrame might tarnish its intention.